### DISCUSSION TITLE
Tool Documentation Site

### DISCUSSION THREAD

**Discussion by rschwietzke** [google/docsy#560] [Show and tell]
We just brought our tool documentation site online: https://xltdoc.xceptance.com/

Just wanted to say thank you to the Docsy folks and show a Docsy example of some production documentation. The source code is available at https://github.com/Xceptance/xlt-documentation. We are looking for comments, suggestions, PRs, and issues. If you find something you like, such as a style or template change, grab it. All under Apache License 2.0.

**Comment by emckean** [comment]
Awesome! Thank you for sharing!

**Comment by LisaFC** [comment]
This is great!

**Comment by prdsoftware** [comment]
I like the way you have a separate page for your release history.  That's a nice touch and easy to find.

**Comment by LisaFC** [comment]
Could I ask you to add the site to our Examples page? We'd love to have it in our collection! https://www.docsy.dev/docs/examples/

**Reply by rschwietzke** [reply]
It is already on it: https://github.com/google/docsy/blob/master/userguide/content/en/docs/Examples/_index.md

**Reply by LisaFC** [reply]
Excellent (also that will teach me to actually look at the Examples page before telling people to update it....)
